tools/Generator/Xamarin.Forms.Core.json

```diff
@@ -2483,7 +2586,8 @@
         },
         {
           "name": "FlyoutHeader",
-          "defaultValue": "null"
+          "defaultValue": "null",
+          "modelType": "ViewElement"
```

Comment:
Had to force ViewElement here to enable content creation from ViewElement.
By default, FlyoutHeader is obj